src/content/doc-sdk-golang/methods/live_notification.mdx

```diff
@@ -39,7 +39,30 @@ db.LiveNotifications[UUID string](queryUuid)
 ### Example usage
 
 ```go title="Example"
-await using var liveQuery = db.LiveNotifications[Person](queryUuid);
+package main
 
-// Consume the live query...
-```
\ No newline at end of file
+import (
+    "fmt"
+    "github.com/surrealdb/surrealdb.go"
+    "github.com/surrealdb/surrealdb.go/pkg/connection"
+)
+
+func main() {
+    // Assuming db is an instance of your database connection
+    // and queryUuid is from a previous Live() call
+    queryUuid := "your-query-uuid-here"
+
+    notifications, err := db.LiveNotifications(queryUuid)
+    if err != nil {
+        fmt.Println("Error setting up notifications:", err)
+        return
+    }
+
+    // Listen for notifications in a goroutine
+    go func() {
+        for notification := range notifications {
+            // Process each notification as it arrives
+            fmt.Printf("Received notification: %+v\n", notification)
+        }
+    }()
+}
```

Comment:
You'll need to wait on something here. Else the program will finish irregardless of the go routine. I suggest waiting for a ctrl-C.

Add the `os/signal` package to the import list. Then add the following code to the end of the program:

```
   sigs := make(chan os.Signal, 1)
   signal.Notify(sigs, syscall.SIGINT, syscall.SIGTERM)
   <-sigs
``` 

That waits for a ctrl-c.